admin/local/docker/compose/docker-compose.yml

```diff
@@ -171,12 +171,13 @@ services:
         << : *worker-xrootd
         command: >
           entrypoint --log-level DEBUG worker-xrootd
-          --db-uri mysql://qsmaster@127.0.0.1:3306?socket=/qserv/mariadb/run/mysqld.sock
-          --db-admin-uri mysql://root:CHANGEME@127.0.0.1:3306?socket=/qserv/mariadb/run/mysqld.sock
+          --db-uri mysql://qsmaster@127.0.0.1:3306?socket={{db_socket}}
+          --db-admin-uri mysql://root:CHANGEME@127.0.0.1:3306?socket={{db_socket}}
```

Comment:
--db-uri is used for more than one thing; entrypoint blocks until it can connect to the db at the uri. Later it's used in the czar cfg file. The latter can use the socket, but the former requires a host & port right now. I created DM-33455 to add socket support there.

Reply:
I'm sorry, I might not be sufficiently specific in my prior comment. It wasn't about the role of that URI. It was about its format, which I believe should have one of these forms:
```
mysql://<db_user>:<db_pass>@<host>:<port>/<db_name>
mysql://<db_user>:<db_pass>@/<db_name>?unix_socket=<socket_path>
```
These are mutually exclusive. The first form (`host` and `port`) is used for connecting over the TCP protocol. The second one (`socket` file) - is for connecting over the local UNIX socket.